### ISSUE TITLE
Raid Completion Difference

### ISSUE THREAD

**Issue by macnode** [TrackerNetwork/DestinyStatus#38]
As of this moment, if I go to the "checklist" and add up all the raids in the 'Raids' section the result is '169'.

My 'RAIDS INTRODUCTION' grim card at bungie.net shows 165 completions.

XB1 GT: L0r3

Minor bug.

**Comment by iBotPeaches** [comment]
I looked into this (sorry late), and our method of relying on the aggregate query seems better than trusting the grimoire card. If my memory is correct, when Dark Below first launched completions of that raid did **not** increase the Raids Introduction card. Only in a subsequent patch update did this counter begin increasing, so this may explain the difference.

I ran my own account as a test. The grimoire card shows 335 completions, while the aggregate count shows 411. This is a 76 raid difference, which is probably due to the reason explained above.

So the inconsistency came from a bug on Bungie's part, nothing we can really do about this.
